tests/tests_suite_test.go

```diff
@@ -352,6 +388,13 @@ func getSsp() *sspv1beta1.SSP {
 	return foundSsp
 }
 
+func getTemplateValidatorDeployment() *apps.Deployment {
```

Comment:
This function is only used from a single file, it could be moved there. But you don't have to.